doc/src/conf.py

```diff
@@ -81,6 +81,23 @@
     "tutorial/matrices": "../tutorials/intro-tutorial/matrices.html",
     "tutorial/manipulation": "../tutorials/intro-tutorial/manipulation.html",
 
+    "modules/physics/vector/index": "../explanation/Modules/Physics/vectors/index.html",
```

Comment:
I think we should keep all directory names lower case, as that is standard in most (maybe all) of SymPy's code.